javascript/app-model-exposure/src/components/EffectScatterplot.tsx

```diff
@@ -121,9 +121,10 @@ function useOverlay(
 		const refutationColors = [
 			effect,
 			inactiveColor,
-			colors(1).hex(),
-			colors(1).hex(),
+			colors(0).hex(),
```

Comment:
With the newer thematic versions you can use theme.nearest(‘green’) to ensure you always get green (or whatever css color you want) even if the theme changes